kratos/python/add_geometrical_utilities_to_python.cpp

```diff
@@ -17,6 +17,7 @@
 
 // Project includes
 #include "includes/define_python.h"
+#include <pybind11/numpy.h>
```

Comment:
This is an external dependency

Reply:
no, it is only the interface to pybind. We already include it with Kratos together with the pybind code

Reply:
I mean it is not a "Project includes", but "External includes"